GVFS/GVFS.PostIndexChangedHook/GVFS.PostIndexChangedHook.Windows.vcxproj

```diff
@@ -14,7 +14,7 @@
     <ProjectGuid>{24D161E9-D1F0-4299-BBD3-5D940BEDD535}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>GVFSPostIndexChangedHook</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.10240.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>

```

Comment:
16299 is RS3. This should be OK?

Reply:
> 16299 is RS3. This should be OK?

We still need to support RS1.  If we switch to RS3 are we risking the newer Windows SDK calling function(s) that aren't present in RS1?

Reply:
>  If we switch to RS3 are we risking the newer Windows SDK calling function(s) that aren't present in RS1?

*Full disclosure:*  I'm not sure how large of a risk this is, or what the official guidance is on what Windows version to target.  Historically the concern has been that we might accidentally break RS1/Server2016.

Reply:
Another option is to install the 10240 SDK manually if you only have VS2019:

https://developer.microsoft.com/en-us/windows/downloads/sdk-archive

It might be interesting to see what the requirements are around supporting specific versions of Windows.